api/src/express-server.ts

```diff
@@ -25,6 +25,11 @@ export class ExpressServer {
     this.app.use(cors())
     this.app.use(helmet())
 
+    this.app.use((req, res, next) => {
+      res.setHeader('Content-Security-Policy', "script-src 'self' 'unsafe-inline'");
+      next();
+    });
```

Comment:
Nice. Maybe we can just set this if the build `process.env.NODE_ENV !== 'production'`

Reply:
Ahh, cause in prod we don't need the explorer